tests/tensor_search/test_bulk_search.py

```diff
@@ -0,0 +1,1015 @@
+import math
```

Comment:
test suggestions
- multiple searchable attributes
- wieghted queries
- boosting fields

Reply:
Yes, Testing is not done.